app/assets/stylesheets/partials/_pagination.scss

```diff
@@ -0,0 +1,47 @@
+
+.ceds-pagination {}
+
+.ceds-pagination__summary {
```

Comment:
```
.ceds-pagination {
  &__summary {
    @extend .govuk-grid-column-one-half;
  }
}
```
Generay this allow you to DRY and not make silly typos in class names - at least in scss file